app/src/main/res/values/attrs.xml

```diff
@@ -141,4 +141,11 @@
         <attr name="showCloseIcon" format="boolean" />
     </declare-styleable>
 
+    <declare-styleable name="AttachmentDetailsView">
+        <attr name="displayStyle" format="enum">
+            <enum name="chip" value="0" />
+            <enum name="bottomSheet" value="1" />
```

Comment:
Maybe "small vs large" instead of "chip vs bottomSheet"

Reply:
After team agreement : we keep chip vs bottomsheet